src/components/Calculator.vue

```diff
@@ -1,168 +1,320 @@
-        display: none;
-      }
+                    this.result();
+
+                    const result = eval(this.calc_screen).toString();
+                    this.history.push(this.calc_screen + '=' + result);
+                    this.first_argument = result;
+                    this.lastSecond_argument = this.second_argument;
+                    this.second_argument = '';
+
+                    this.lastOperation = this.operation;
+                    this.operation = '';
+
+                    this.result();
+                } catch (e) {
+                    this.error = e.message;
+                }
+            },
+
+            isValueExist(valueToSearchIn, regExp) {
+
+                let result = valueToSearchIn.search(regExp);
+                return result;
+            },
+
+            result() {
+                this.calc_screen = this.first_argument + this.operation + this.second_argument;
+            }
+        }
     }
+</script>
 
-    .standard-block {
-      display: inline-grid;
-      grid-template-columns: repeat(4, 1fr);
-      grid-template-rows: repeat(5, 45px);
-      grid-gap: $grid-gap-wide-screen;
-      width: 60%;
-      margin-top: 10px;
-
-      @media screen and (max-width: 620px) {
-        grid-gap: $grid-gap-mobile;
-        width: 100%;
-      }
+<style scoped lang="scss">
+    @import '../assets/variables';
+
+
+    .environment-block {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        height: 80px;
+        margin-top: 60px;
+        padding: 0 $environment-block-padding-big-screen;
+        border-bottom: 1px solid #ebebeb;
+        color: #4f4f4f;
+        background-color: #fbfbfb;
+
+        @media screen and (max-width: 620px) {
+            padding: 0 $environment-block-padding-small-screen;
+        }
+
+        .change-calc-view {
+            width: 120px;
+            height: 25px;
+            border: none;
+            border-radius: 3px;
+            color: #fbfbfb;
+            background-color: #ff7c1f;
+        }
     }
 
-    .calc_buttons button {
-      padding: 0;
-      border-style: none;
-      border-radius: 4px;
-      font-size: 1.1em;
-      font-weight: 400;
-
-      &:hover {
-        cursor: pointer;
-        filter: brightness(95%);
-      }
-
-      &:focus {
-        outline: 0;
-      }
-
-      &.btn-grey {
-        background-color: $grey-color;
-      }
-
-      &.btn-light {
-        background-color: $light-color;
-      }
-
-      &.btn-blue {
-        background-color: $blue-color;
-      }
+    .calculator {
+        max-width: 500px;
+        margin: 0 auto;
+        padding: 20px;
+
+        @media screen and (max-height: 400px) {
+            overflow-y: scroll;
+        }
+
+        .calc-screen {
+            padding-right: 10px;
+            border: solid 2px #4d90fd;
+            border-radius: 4px;
+            text-align: right;
+
+            strong {
```

Comment:
it is better not to match `tag`, use `class` and add style `bold`. `strong` is not bad, rather good, but only if you want to make it bold. But you want more

Reply:
added class to strong